<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInsertInfo</class>
 <widget class="QDialog" name="DialogInsertInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Данные о сотруднике</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_genInfo">
      <attribute name="title">
       <string>Общая информация</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_lastname">
           <property name="text">
            <string>Фамилия</string>
           </property>
           <property name="buddy">
            <cstring></cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_lastname"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_surname">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_surname"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_department">
           <property name="text">
            <string>Отдел</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edit_department"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_position">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Должность</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edit_position"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_hireDate">
           <property name="text">
            <string>Дата приема на работу</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDateEdit" name="edit_hireDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_passportInfo">
      <attribute name="title">
       <string>Паспортные данные</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_passportSeries">
           <property name="text">
            <string>Серия</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_passportSeries"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_passportNumber">
           <property name="text">
            <string>Номер</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_passportNumber"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_passportIssuanceDate">
           <property name="text">
            <string>Дата выдачи</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="edit_issuanceDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_issuanceAgencyName">
           <property name="text">
            <string>Выдавший орган</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edit_issuanceDepartmentName"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_issuanceAgencyCode">
           <property name="text">
            <string>Код подразделения</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edit_issuanceDepartmentCode"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_validityDate">
           <property name="text">
            <string>Срок действия</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDateEdit" name="edit_validityDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_otherDocumentsInfo">
      <attribute name="title">
       <string>Другие документы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_Inn">
           <property name="text">
            <string>ИНН</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_INN"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_Snils">
           <property name="text">
            <string>СНИЛС</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_SNILS"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_medPolicyNumber">
           <property name="text">
            <string>Номер мед. полиса</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_medicalPolicyNumber"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_additionalInfo">
      <attribute name="title">
       <string>Дополнительная информация</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_birthDate">
           <property name="text">
            <string>Дата рождения</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="edit_dateOfBirth">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_BirthPlace">
           <property name="text">
            <string>Место рождения</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_placeOfBirth"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_citizenship">
           <property name="text">
            <string>Гражданство</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_citizenship"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_nationality">
           <property name="text">
            <string>Национальность</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edit_nationality"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_maritalStatus">
           <property name="text">
            <string>Семейное положение</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edit_maritialStatus"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInsertInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInsertInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
